server/py/services/api/tests/unit/utils/projects/test_follower_member.py

```diff
@@ -468,7 +468,7 @@ async def test_list_project_summaries_fails_to_list_pipeline_runs(
     )
     framework.utils.singletons.db.get_db().get_project_resources_counters = (
         unittest.mock.AsyncMock(
-            return_value=tuple({project_name: i} for i in range(12))
+            return_value=tuple({project_name: i} for i in range(15))
```

Comment:
What is this magic number?
It seems like it should depend on the number of `<>_count` properties in ProjectSummary, but the number doesn't quite match. 
If it is this, maybe we can calculate this dynamically to be resilient for future changes, e.g.
```
num_count_properties = sum(1 for field_name in ProjectSummary.__fields__ if field_name.endswith('_count'))
```

Reply:
This is meant to represent only the project counters out of the project summary parameters, minus the pipeline counters and project name (`project_to_recent_completed_pipelines_count`, `project_to_recent_failed_pipelines_count`, `project_to_running_pipelines_count` and `project_name`), if you count it all combine it does match.